superset/translations/pt/LC_MESSAGES/message.po

```diff
@@ -0,0 +1,4181 @@
+msgid "Open in SQL Editor"
+msgstr "Abrir no Editor SQL"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:133
+msgid "view results"
+msgstr "ver resultados"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:136
+msgid "Data preview"
+msgstr "Pré-visualização de dados"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:176
+msgid "Visualize the data out of this query"
+msgstr "Visualize os dados desta query"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:182
+msgid "Overwrite text in editor with a query on this table"
+msgstr "Substitua texto no editor com uma query nesta tabela"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:188
+msgid "Run query in a new tab"
+msgstr "Executar a query em nova aba"
+
+#: superset/assets/javascripts/SqlLab/components/QueryTable.jsx:193
+msgid "Remove query from log"
+msgstr "Remover query do histórico"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:67
+msgid ".CSV"
+msgstr ".CSV"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:78
+#: superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx:242
+#: superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx:281
+msgid "Visualize"
+msgstr "Visualize"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:162
+#: superset/connectors/sqla/views.py:84 superset/connectors/sqla/views.py:134
+#: superset/connectors/sqla/views.py:214 superset/views/core.py:389
+msgid "Table"
+msgstr "Tabela"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:162
+msgid "was created"
+msgstr "foi criado"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:169
+msgid "Query in a new tab"
+msgstr "Query numa nova aba"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:210
+msgid "Fetch data preview"
+msgstr "Obter pré-visualização de dados"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:230
+#, fuzzy
+msgid "Track Job"
+msgstr "Acompanhar trabalho"
+
+#: superset/assets/javascripts/SqlLab/components/ResultSet.jsx:236
+msgid "Loading..."
+msgstr "A carregar..."
+
+#: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx:19
+#, fuzzy
+msgid "Run Selected Query"
+msgstr "Executar a query selecionada"
+
+#: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx:19
+#, fuzzy
+msgid "Run Query"
+msgstr "Executar query"
+
+#: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx:22
+#, fuzzy
+msgid "Run query asynchronously"
+msgstr "Execute a query de forma assíncrona"
+
+#: superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx:57
+msgid "Stop"
+msgstr "Parar"
+
+#: superset/assets/javascripts/SqlLab/components/SaveQuery.jsx:16
+msgid "Undefined"
+msgstr "Indefinido"
+
+#: superset/assets/javascripts/SqlLab/components/SaveQuery.jsx:66
+#: superset/views/sql_lab.py:53
+#, fuzzy
+msgid "Label"
+msgstr "Label"
```

Comment:
"Etiqueta"? There are some more messages below where "Label" is translated as "Etiqueta", and some others were it's left unstranslated.